tools/tests/com.liferay.ide.xml.search.ui.tests/src/com/liferay/ide/xml/search/ui/tests/LiferayHookXmlTests.java

```diff
@@ -341,21 +344,10 @@ public void testPortalPropertiesValidation() throws Exception
         buildAndValidate( descriptorFile );
     }
 
-    // TODO
+    @Test
     public void testServiceTypeAndServiceImpl() throws Exception
     {
-        // test service-type and service-impl together, because validation of service-impl relies on service-type
-
-        // Service type
-        // Foo "Type not found"
-        // com.liferay.ide.tests.OrphanClass "Not a interface"
-        // com.liferay.ide.tests.OrphanInterface "Should match *Service"
-        // com.liferay.ide.tests.FooService, correct
-
-        // Service Imple
-        // Foo "Type not found"
-        // OrphanClass "Type Hierarchy incorrect"
-        // FooServiceWrapperImpl correct
+    	testServiceTypeAndServiceImplValidation();
```

Comment:
No need to just call the function you can put the functions contents directly here.